bin/iuf-helm-upload

```diff
@@ -26,8 +26,8 @@
 SCRIPT_DIR=$(dirname "${BASH_SOURCE}")
 
 usage(){
-    echo "Usage: $0 JSON_IUF_HELM_META"
-    echo "Uploads Helm chart(s) from the location(s) given by the JSON_RPM_HELM_META structure to the Nexus registry."
+    echo "INFO Usage: $0 JSON_IUF_HELM_META"
```

Comment:
i don't think we need to add INFO the usage messages.  i don't think you should normally get these msgs from IUF and if you did, no prefix would make them debug which is probably fine